GVRf/Framework/framework/src/main/jni/objects/textures/astc_transparency.cpp

```diff
@@ -193,7 +193,7 @@ void decode_block(uint32_t block_mode, decoded_block_t *decoded_block)
     {
         // The 'selector' is bits 7 & 8 which distinguish which row
         // of the table applies to retrieve the values we're looking for.
-        selector == (block_mode >> 7) & 0x3;
+        selector = (block_mode >> 7) & 0x3;
```

Comment:
@thomasflynn The selector I am sure was supposed to be assigned to.

Reply:
agreed.  not sure how that slipped past my tests.  good catch!